.github/workflows/build.yml

```diff
@@ -0,0 +1,91 @@
+name: Docker build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: "0 1 * * *" # Daily “At 01:00” UTC
+  workflow_dispatch:
```

Comment:
Not sure if there's a preference on how something like this would get triggered on a schedule? Ideally would like to have the status checks broadcast on Slack to improve visibility, so happy to move in whatever direction makes that easiest with our current devops stack

Reply:
This looks ok to me. cc @raydouglass for additional review

FWIW this post might be relevant for your slack notification needs: https://github.blog/changelog/2022-12-06-github-actions-workflow-notifications-in-slack-and-microsoft-teams/